style.css

```
@@ -1,11 +1,224 @@
-/* CSS files add styling rules to your content */
+/* ====================
+    GLOBAL STUFF
+   ====================*/
+
+* {
+  box-sizing: border-box;
+}
 
 body {
-  font-family: helvetica, arial, sans-serif;
+  background-color: #fdf6dd;
+  font-family: "Source Sans Pro", arial, sans-serif;
   margin: 2em;
 }
 
+.container {
+  color: #000000;
+  margin: 0 auto;
+  padding: 0;
+  position: relative;
+  width: 800px;
+}
+
+.container div {
+  background-color: #fff;
+  border-radius: 15px;
+  margin: 15px;
+  padding: 20px;
+}
+
 h1 {
-  font-style: italic;
-  color: #373fff;
+  font-size: 40px;
+  margin: 0;
+  padding: 10px 0;
+}
+
+h2 {
+  font-size: 36px;
+  margin: 0 0 10px 0;
+  padding: 10px;
+}
+
+h3 {
+  font-size: 30px;
+  margin: 0 0 10px 0;
+  padding: 10px;
+}
+
+ul {
+  margin: 0 45px;
+  padding: 0;
+}
+
+h4 {
+  margin: 0;
+  padding: 5px;
+}
+
+p {
+  color: black;
+  font-size: 14px;
+  margin: 2px;
+  padding: 5px;
+  margin-left: 20px;
```

Combine this CSS declaration as you have done elsewhere:
```suggestion
  margin: 2px 0 0 20px;
  padding: 5px;
```